healpy/fitsfunc.py

```diff
@@ -347,9 +338,8 @@ def read_map(
       the header number to look at (start at 0)
     h : bool, optional
       If True, return also the header. Default: False.
-    verbose : bool, optional
-      If True, print a number of diagnostic messages, call hp.disable_warnings() to
-      disable warnings for all functions.
+    verbose : bool, not supported anymore
```

Comment:
Deprecate this so that it can be removed in a future release. Astropy's utils module has some features for deprecating arguments.

Reply:
this would generate warnings. isn't it acceptable to ask users to go through their code and remove all `verbose` arguments? it seems a quick fix that can be just done once (and will need to be done in the future).

Reply:
A deprecation warning is least obtrusive way to tell users to do that.

Reply:
https://docs.astropy.org/en/stable/api/astropy.utils.decorators.deprecated_renamed_argument.html#astropy.utils.decorators.deprecated_renamed_argument

Reply:
ok